wp-parsely.php

```diff
@@ -152,19 +152,33 @@ public function initialize_settings() {
                            Parsely::MENU_SLUG, 'optional_settings',
                            $field_args);
 
-        // Use top-level cats
+         // Use top-level cats
         $h = 'wp-parsely will use the first category assigned to a post. ' .
              'With this option selected, if you post a story to News > ' .
              'National > Florida, wp-parsely will use the "News" for the ' .
-             'section name instead of "Florida".';
+             'section name in your dashboard instead of "Florida".';
         add_settings_field('use_top_level_cats',
-                           'Use Top-Level Categories <div class="help-icons"></div>',
+                           'Use Top-Level Categories for Section <div class="help-icons"></div>',
                            array($this, 'print_binary_radio_tag'),
                            Parsely::MENU_SLUG, 'optional_settings',
                            array('option_key' => 'use_top_level_cats',
                                  'help_text' => $h,
                                  'requires_recrawl' => true));
 
+        // Allow use of custom taxonomy to populate articleSection in parselyPage; defaults to category
+        $h = 'By default, the section value in your Parse.ly dashboard maps to a post\'s category. ' .
+             'You can optionally choose a custom taxonomy, if you\'ve created one, to ' .
+             'populate the section value instead. <br>';
+        add_settings_field('custom_taxonomy_section',
+                           'Use Custom Taxonomy for Section  <div class="help-icons"></div>',
+                           array($this, 'print_select_tag'),
+                           Parsely::MENU_SLUG, 'optional_settings',
+                           array('option_key' => 'custom_taxonomy_section',
+                                 'help_text' => $h,
+                                 // filter Wordpress taxonomies under the hood that should not appear in dropdown
+                                 'select_options' => array_diff(get_taxonomies(), array('post_tag', 'nav_menu', 'link_category', 'post_format')),
+                                 'requires_recrawl' => false));
```

Comment:
might want to flag requires_recrawl as true here since if they change it in an existing install it will require a recrawl